src/components/header/renderer.tsx

```diff
@@ -479,6 +525,64 @@ class Header extends React.Component<Props, State> {
       </div>
     );
 
+    const copiedClass = 'copied' + (this.state.copied ? ' visible' : '');
+
+    const shareContent = (
+      <div className="share-content">
+        <h2>Share</h2>
+        <p>We pack the Vega or Vega-Lite specification and an encoded string in the URL.</p>
+        <p>We use and LZ-based compression algorithm and preserve indentation, newlines, and other whitespace.</p>
+        <div className="user-pref">
+          <label>
+            Link opens visualization in fullscreen:
+            <input
+              type="checkbox"
+              defaultChecked={this.state.fullscreen}
+              name="fullscreen"
+              onChange={this.handleCheck.bind(this)}
+            />
+          </label>
+        </div>
+        <div className="exported-url">
+          <span ref="wrapperURL">
+            <a ref="exportedURL" href={this.state.generatedURL} target="_blank">
+              {this.state.generatedURL}
+            </a>
+          </span>
+        </div>
+        <div className="copy-clipboard">
+          <Clipboard
+            className="copy-icon"
+            data-clipboard-text={this.state.generatedURL}
+            onSuccess={this.onCopy.bind(this)}
+          >
+            <div className="copy-button">
+              <Copy size={20} /> <span>Copy to Clipboard</span>
+            </div>
+          </Clipboard>
+          <div className={copiedClass}>Copied!</div>
+        </div>
+        <div className="byte-counter">
+          Bytes: {this.state.generatedURL.length}{' '}
+          <span className="warning">
+            {this.state.generatedURL.length > 2083 ? (
+              <span>
+                Warning:{' '}
```

Comment:
add space between : and content

Reply:
There is a space in the final rendering.